FAQ.md

```diff
@@ -667,6 +667,10 @@ TiKV supports calling the interface separately. Theoretically, you can take an i
 
 This is because the disk space is not enough. You need to add nodes or enlarge the disk space.
 
+#### Why does the OOM (Out of Memory) error occur frequently in TiKV?
+
+TiKV uses the memory mainly for the block-cache of RocksDB, which is 40% of the system memory size by default. When the OOM error occurs frequently in TiKV, you should check if the value of `block-cache-size` is set too high. In addition, when multiple TiKV instances are deployed on a single machine, you need to explicitly configure the parameter to prevent multiple instances from using too much system memory that results in the OOM error. 
```

Comment:
```suggestion
The memory usage of TiKV mainly comes from the block-cache of RocksDB, which is 40% of the system memory size by default. When the OOM error occurs frequently in TiKV, you should check whether the value of `block-cache-size` is set too high. In addition, when multiple TiKV instances are deployed on a single machine, you need to explicitly configure the parameter to prevent multiple instances from using too much system memory that results in the OOM error. 
```

Reply:
Updated. PTAL again.